README.md

```diff
@@ -11,6 +11,12 @@
 npm install marked --save
 ```
 
+or if you want to use the `marked` CLI tool:
```

Comment:
maybe worth noting that global install isn't necessary when using npm run-scripts?